peas/sensors.py

```diff
@@ -60,52 +57,9 @@ def __init__(self, auto_detect=False, *args, **kwargs):
                     'port': port,
                 }
 
-    def _auto_detect_port(self, port):
-        """Determines the type of Arduino attached to the port.
-
-        Returns: tuple of (sensor_name, serial_reader) if able to determine a name,
-            else returns None.
-        """
-        if os.path.exists(port):
-            self.logger.debug('Port {} exists', port)
-        elif '://' in port:
-            self.logger.debug('Port {} may have a PySerial handler installed; testing.', port)
-        else:
-            self.logger.debug('Port {} not found', port)
-            return None
-        try:
-            serial_reader = self._connect_serial(port)
-            if serial_reader is None:
-                return None
-        except Exception:
-            self.logger.debug('Failed to connect on {}', port)
-            return None
-        # Try to get a reading with a full line. Sometimes the first line is
-        # partial, so we need to read a second line to find a line with the
-        # name of the board in the reading.
-        num_tries = 5
-        for _ in range(num_tries):
-            try:
-                self.logger.debug('Reading from {}', port)
-                data = self._get_parsed_reading(serial_reader)
-                if data and len(data) == 2 and 'name' in data[1]:
-                    parsed = data[1]
-                    sensor_name = parsed['name']
-                    self.logger.debug('Found name {}', sensor_name)
-                    result = (sensor_name, serial_reader)
-                    serial_reader = None
-                    return result
-                self.logger.debug("'name' not found in reading")
-            except Exception:
-                pass
-            finally:
-                if serial_reader:
-                    serial_reader.disconnect()
-        return None
-
     def _connect_serial(self, port):
         self.logger.debug('Attempting to connect to serial port: {}'.format(port))
-        serial_reader = SerialData(port=port)
+        serial_reader = SerialData(port=port, baudrate=9600)
```

Comment:
I'm guessing it's not worth making this a variable?

Reply:
We should probably address this along with #181 (i.e. even without the port being specified in the yaml file, we could have other serial params there, including baudrate).